README.md

```diff
@@ -27,8 +22,30 @@ Supported environments
 
 *1 The host's Linux Kernel must be v4.9 or later which supports `rmapbt` and `reflink`, if you use xfs filesystem with an official docker image.
 
-Features
---------
+Docker images are available on [ghcr.io](https://github.com/orgs/topolvm/packages).  
+**Please note that [the images on quay.io](https://quay.io/organization/topolvm) are currently deprecated and will no longer be available in the future.**
+
+Getting started
+---------------
+
+A demonstration of TopoLVM running on [kind (Kubernetes IN Docker)][kind] is available at [example](example/) directory.
+
+For production deployments, see [deploy/README.md](./deploy/README.md).
+
+User manual is at [docs/user-manual.md](docs/user-manual.md).
+
+_Deprecated: If you want to use TopoLVM on [Rancher/RKE](https://rancher.com/docs/rke/latest/en/), see [docs/deprecated/rancher.md](docs/deprecated/rancher.md)._
+
+Contributing
+------------
+
+Our project welcomes contributions from any member of our community. To get
+started contributing, please see our [Contributor Guide](CONTRIBUTING.md).
```

Comment:
```suggestion
started contributing, please see our [Contributing Guide](CONTRIBUTING.md).
```

Reply:
fixed. (it is from the template though :)